addons/compat_spe/SPE_Tanks.hpp

```diff
@@ -354,9 +390,153 @@ class SPE_FR_M4A0_75_mid_Command: SPE_FR_M4A0_75_mid {
     };
 };
 class SPE_FR_M4A0_75_mid_Command_DVL: SPE_FR_M4A0_75_mid {
+    editorSubCategory = "SPE_EdSubcat_Tanks_DLV";
+    hasDriver = -1;
     vehicleClass = "Armored_DLV";
+};
+
+class SPE_M4A0_105: SPE_M4A0_75 {
```

Comment:
```suggestion
class SPE_M4A0_105: SPE_M4A0_75 {};
```

Reply:
Good spot, but we actually don't need to the inheritence at all here!
I'll update it to be `class SPE_M4A0_105;`